src/schema/objects/metadata.yaml

```diff
@@ -356,6 +374,47 @@ BrainLocation:
     or layer to refer to layer-specific gene expression,
     which can also tie up with laminar fMRI.
   type: string
+CalibrationList:
+  name: CalibrationList
+  display_name: Calibration List
+  description: |
+    List of lists including information for each calibration.
+    This list includes the calibration type, recorded eye,
+    maximal calibration error, average calibration error,
+    and time relative to the first event of the event file.
+  type: object
+  items:
+    type: object
```

Comment:
If it's a list of lists, then the type should be "array", rather than "object" (which is more like a dictionary).